src/main/scala/com/amazon/deequ/analyzers/Distance.scala

```diff
@@ -0,0 +1,90 @@
+/**
+ * Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may not
+ * use this file except in compliance with the License. A copy of the License
+ * is located at
+ *
+ *     http://aws.amazon.com/apache2.0/
+ *
+ * or in the "license" file accompanying this file. This file is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ *
+ */
+
+package com.amazon.deequ.analyzers
+
+case class CategoricalBucket(value: String, count: Long)
+
+case class Categorical(buckets: List[CategoricalBucket])
```

Comment:
Why do we use a list of key-value pairs and not a hashmap? It seems this is used like a hashmap later on

Reply:
Again, I would actually like to design it as a hashmap if I could, but for historic reasons in our codebases, and because we are going to use these data structures as input later, I would be inclined to keep it as it is.